Tensile/Common.py

```diff
@@ -2390,6 +2390,10 @@ def assignGlobalParameters( config ):
     else:
       tPrint(3, " %24s: %8s (unspecified)" % (key, defaultValue))
 
+  if "KeepBuildTmp" in config:
+    globalParameters["KeepBuildTmp"] = config["KeepBuildTmp"] 
```

Comment:
Don't we need to initialize globalParameters["KeepBuildTmp"]?
I thought we needed to add some init code at the beginning of Common.py (as other globalParameters).

Reply:
For this feature it is both a command line option and a config file option. What this means is that setting a default in common will break the ability to fall back on a command line option if it isn't set in the config file. If we want to only specify this in a config file we could set a default as you suggested. An example from your config file would be PrintSolutionRejectionReason. This has no command line option so it needs a default in common.